.github/workflows/ci-c-warnings.yml

```diff
@@ -53,6 +53,18 @@ jobs:
       - name: Enable nodiscard
         if: ${{ matrix.case == 0}}
         run: echo "CMAKE_AVIF_FLAGS=\"-DAVIF_ENABLE_NODISCARD=ON\""  >> $GITHUB_ENV
+      - name: Use stdbool for AVIF_TRUE/AVIF_FALSE
+        if: ${{ matrix.case == 1}}
+        run: |
+          sed -i 's/#include <stdint.h>/#include <stdint.h>\n#include <stdbool.h>/' include/avif/avif.h
+          sed -i 's/typedef int avifBool;/typedef bool avifBool;/' include/avif/avif.h
+          sed -i 's/#define AVIF_TRUE 1/#define AVIF_TRUE true/' include/avif/avif.h
+          sed -i 's/#define AVIF_FALSE 0/#define AVIF_FALSE false/' include/avif/avif.h
+      - name: Use enums for AVIF_TRUE/AVIF_FALSE
```

Comment:
Do we also need this alternative definition? Does stdbool not catch all the bugs?

Reply:
Nope, cf the other PR: both found different bugs .....